system/mp/mpcomm.cpp

```diff
@@ -756,8 +760,7 @@ protected: friend class CMPPacketReader;
                 // loop with short wait time and release CS to allow other side to proceed
                 StringBuffer epStr;
                 unsigned startMs = msTick();
-
-                unsigned loopCnt = ((CONNECT_READ_TIMEOUT / retrycount) / CONNECT_TIMEOUT_INTERVAL) + 1;
+                unsigned loopCnt = CONNECT_TIMEOUT_LOOPCNT;
```

Comment:
>This makes the overall timeout for the connection a lot more that it was before.

.. yes, because the retrycount was increased from 10 to 20. 

>Should we make it just based on the original value of retrycount.

```
unsigned loopCnt = ((CONNECT_READ_TIMEOUT / CONNECT_RETRYCOUNT) / CONNECT_TIMEOUT_INTERVAL) + 1;
```

Then each of these inner loops (i.e. each retrycount) is going to take [max] 10 secs...
Ok there's 20 retries vs original 10, but that's only [max] 200 secs overall.
NB: It used to be more, progressively, as retryCount decreased, but that was an accidental artefact.

The original [max] timeout was 1800s  (10 * 180s)


Why was retrycount upped? If it was reverted to 10 and each retry had this PR's fixed loop cnt (not based on retrycount) then it would be equivalent to what was there originally, wouldn't it?